db/bigtable.go

```diff
@@ -61,6 +72,19 @@ type Bigtable struct {
 }
 
 func InitBigtable(project, instance, chainId, redisAddress string) (*Bigtable, error) {
+
+	if utils.Config.Bigtable.Emulator {
+
+		if utils.Config.Bigtable.EmulatorHost == "" {
+			utils.Config.Bigtable.EmulatorHost = "127.0.0.1"
+		}
+		logger.Infof("using emulated local bigtable environment, setting BIGTABLE_EMULATOR_HOST env variable to %s:%d", utils.Config.Bigtable.EmulatorHost, utils.Config.Bigtable.EmulatorPort)
```

Comment:
nitpick, could be a remote host

Reply:
remote hosts can be specified using the new `EmulatorHost` field in the config.